.github/workflows/dart.yml

```diff
@@ -0,0 +1,22 @@
+name: Dart CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    container:
+      image:  google/dart:latest
```

Comment:
for some reason flutter sdk isn't installed on this container or if it is its not in the PATH so you'll need to add the flutter sdk. [This](https://gist.github.com/Kthulu120/59639e3a81dc8e61f37c78177d39ac20) is the Dart Action that I use to automatically produce builds for flutter on master branch. You'll need to change it to all branches besides that as long as the tests pass this config will work. Just a option.